docs/en/how-fluent-works.md

```diff
@@ -0,0 +1,38 @@
+# How Fluent works
+
+In older versions of Fluent, localisation was stored on a single table row.
+However, versions 4.0.0 and onwards will store localisations for any model
+separately to the main table in a similar table suffixed with `_Localised`.
+
+For example, a versioned SiteTree object will have these tables:
+
+ - SiteTree
+ - SiteTree_Live
+ - SiteTree_Versions
+ - SiteTree_Localised (joined to SiteTree)
+ - SiteTree_Localised_Live (joined to SiteTree_Live)
+ - SiteTree_Localised_Versions (joined to SiteTree_Versions)
+
+The unique key for the _Localised table is `RecordID`, `Locale`
```

Comment:
Can we please escape the `_`_ in these examples so it doesn't break the markdown rendering